Source/WebKit/UIProcess/ios/PageClientImplIOS.mm

```diff
@@ -741,8 +745,16 @@ - (BOOL)_isHostedInAnotherProcess;
     [contentView() _showPlaybackTargetPicker:hasVideo fromRect:elementRect routeSharingPolicy:policy routingContextUID:contextUID.createNSString().get()];
 }
 
-bool PageClientImpl::handleRunOpenPanel(WebPageProxy*, WebFrameProxy*, const FrameInfoData& frameInfo, API::OpenPanelParameters* parameters, WebOpenPanelResultListenerProxy* listener)
+bool PageClientImpl::handleRunOpenPanel(WebPageProxy* proxy, WebFrameProxy*, const FrameInfoData& frameInfo, API::OpenPanelParameters* parameters, WebOpenPanelResultListenerProxy* listener)
```

Comment:
I would name the parameter "page" rather than "proxy":
- That's how `WebPageProxy` parameters are normally named
- There's also a `WebFrameProxy` here

Reply:
Done